src/routes/+layout.svelte

```diff
@@ -63,8 +62,9 @@
 	)
 
 	$effect(() => {
-		pathname
-		menuTitleIsOpen = makeMenuItemOpenMapping()
+		if (pathname) {
+			menuTitleIsOpen = makeMenuItemOpenMapping()
+		}
```

Comment:
this only alters the variable `menuTitleIsOpen` if `pathname is defined. Probably not desired outcome.